tests/solidity_test_helpers/MockAugur.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity 0.4.17;
+
+
+import 'Controlled.sol';
+import 'libraries/token/ERC20.sol';
+import 'reporting/IUniverse.sol';
+import 'reporting/IMarket.sol';
+import 'reporting/IParticipationToken.sol';
+import 'reporting/IStakeToken.sol';
+import 'reporting/IReputationToken.sol';
+import 'trading/IShareToken.sol';
+import 'trading/Order.sol';
+
+
+// AUDIT/CONSIDER: Is it better that this contract provide generic functions that are limited to whitelisted callers or for it to have many specific functions which have more limited and specific validation?
+contract MockAugur is Controlled {
+    event MarketCreated(address indexed universe, address indexed market, address indexed marketCreator, uint256 marketCreationFee, string extraInfo);
+    event DesignatedReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event ReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event WinningTokensRedeemed(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountRedeemed, uint256 reportingFeesReceived, uint256[] payoutNumerators);
+    event ReportsDisputed(address indexed universe, address indexed disputer, address indexed market, IMarket.ReportingState reportingPhase, uint256 disputeBondAmount);
+    event MarketFinalized(address indexed universe, address indexed market);
+    event UniverseForked(address indexed universe);
+    event OrderCanceled(address indexed universe, address indexed shareToken, address indexed sender, bytes32 orderId, Order.TradeTypes orderType, uint256 tokenRefund, uint256 sharesRefund);
+    event OrderCreated(address indexed universe, address indexed shareToken, address indexed creator, bytes32 orderId, uint256 tradeGroupId);
+    event OrderFilled(address indexed universe, address indexed shareToken, address filler, bytes32 orderId, uint256 numCreatorShares, uint256 numCreatorTokens, uint256 numFillerShares, uint256 numFillerTokens, uint256 settlementFees, uint256 tradeGroupId);
+    event ProceedsClaimed(address indexed universe, address indexed shareToken, address indexed sender, address market, uint256 numShares, uint256 numPayoutTokens, uint256 finalTokenBalance);
+    event UniverseCreated(address indexed parentUniverse, address indexed childUniverse);
+    event TokensTransferred(address indexed universe, address indexed token, address indexed from, address to, uint256 value);
```

Comment:
Seems like we can remove these if they're never used

Reply:
I don't know if they will be used. I'll remove them and add them back if need be